src/views/createQuestionElement.js

```diff
@@ -1,30 +1,38 @@
 'use strict';
 
-import createDOMElement from "../utils/createDOMElement.js";
+import createDOMElement from '../utils/createDOMElement.js';
+import clickAnswerHandler from '../handlers/clickAnswerHandler.js';
 
 const createQuestionElement = (question) => {
-    const container = createDOMElement('div');
-    const title = createDOMElement('h1');
-    title.innerText = question.text;
-    container.appendChild(title);
+  const container = createDOMElement('div');
+  const title = createDOMElement('h1');
+  title.innerText = question.text;
+  container.appendChild(title);
 
-    const answerContainer = createDOMElement('ol');
+  const answerContainer = createDOMElement('ol');
 
-    for (const answerKey in question.answers) {
-        const answer = createAnswerElement(question.answers[answerKey]);
-        answerContainer.appendChild(answer);
-    }
+  for (const answerKey in question.answers) {
+    const answer = createAnswerElement(question.answers[answerKey]);
+    const index = Object.keys(question.answers);
+    answer.dataset.questionIndex = index.indexOf(answerKey); // 0, 1, 2 ...
```

Comment:
This is very smart! Love it. I think it becomes a little bit more readable if you don't assign the `index` variable:

```suggestion
    answer.dataset.questionIndex = Object.keys(question.answers).indexOf(answerKey); // 0, 1, 2 ...
```